src/cases/NgRestTestCase.php

```diff
@@ -300,5 +351,77 @@ public function beforeTearDown()
         $this->userGroupFixture->cleanup();
         $this->userOnlineFixture->cleanup();
         $this->ngrestLogFixture->cleanup();
+        $this->app->db->createCommand()->dropTable('admin_auth')->execute();
+        $this->app->db->createCommand()->dropTable('admin_group_auth')->execute();
+        $this->app->db->createCommand()->dropTable('admin_user_group')->execute();
+    }
+
+    protected function removeApiPermissions()
+    {
+        $this->app->db->createCommand()->delete('admin_auth', ['id' => self::ID_AUTH_API])->execute();
+    }
+
+    protected function resetApiPermissions($create = false, $update = false, $delete = false)
+    {
+        $this->app->db->createCommand()->update('admin_group_auth',
+                                                [
+                                                    'crud_create' => (int)$create,
+                                                    'crud_update' => (int)$update,
+                                                    'crud_delete' => (int)$delete,
+                                                ],
+                                                [
+                                                    'group_id' => self::ID_GROUP_TESTER,
+                                                    'auth_id' => self::ID_AUTH_API,
+                                                ])->execute();
+        return $this;
+    }
+
+    private function setApiPermission($permission, $value)
+    {
+        $this->app->db->createCommand()->update('admin_group_auth',
+                                                [
+                                                    "crud_$permission" => (int)$value,
+                                                ],
+                                                [
+                                                    'group_id' => self::ID_GROUP_TESTER,
+                                                    'auth_id' => self::ID_AUTH_API,
+                                                ])->execute();
+    }
+
+    protected function apiCanCreate($value = true)
```

Comment:
protected methods are available trough inheritance, therefore php doc is required including a @since tag.